native/src/modules/common/components/RemoteContent.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 
-import * as React from 'react'
+import React, { useState, useEffect, type Element } from 'react'
```

Comment:
Only converted this to functional component because I initially thought the work has to be done here...